app/cp_nigeria/views.py

```diff
@@ -206,7 +206,12 @@ def cpn_scenario_create(request, proj_id=None, step_id=STEP_MAPPING["choose_loca
                 qs_report = ImplementationPlanContent.objects.filter(simulation=project.scenario.simulation)
                 if qs_report.exists() and economic_data.has_changed():
                     qs_report.delete()
-            economic_data = economic_data.save()
+            economic_data = economic_data.save(commit=False)
+            # set the initial values for discount and tax
+            economic_data.discount = 0.12
```

Comment:
It can later be changed in the economic parameters view, this just sets the value initially (it's a bit confusing because the fields of the model are filled over multiple views).